guide/14-deep-learning/address_standerdization_using_model_extension.ipynb

```diff
@@ -0,0 +1,967 @@
+{
```

Comment:
<ul><li>Use&nbsp;<strong>ESRI Deep Learning Package (.dlpk) file</strong>&nbsp;everywhere DLPK or Esri deep learning package (.dlpk file) or DLPK (Deep Learning Package) is written</li><li>Similarly Use&nbsp;<strong>ESRI Model Definition (.emd) file&nbsp;</strong>everywhere&nbsp;<strong>emd</strong>&nbsp;is written.</li><li>Explain the term text model its not clear - But what if you encounter a&nbsp;<strong>Text model</strong>&nbsp;or&nbsp;<strong>LLM</strong>&nbsp;that isn’t included in the existing learning module?</li></ul><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/2112/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Reply:
<p>Fixed</p>